guide/registration.md

```diff
@@ -0,0 +1,17 @@
+## User Handbook
+### Registration
+
+To be able to post, and interact with a Flarum powered community, you'll first need to register yourself a user account. The registration process could not be easier, and takes only a few moments of your time.
+
+To sign up, click the "Sign Up" button at the top right of the screen to be presented with the sign up form, like the animation below:
```

Comment:
Completed, explained about SSO options on registration, and added a screenshot on the registration page.